### PR TITLE
golang serving and client trust behavior

### PR DESCRIPTION
**Pull request by deads2k**

### CODE REVIEW

File: pkg/crypto/cert_accept_reject_test.go [hunk 1]
```diff
@@ -0,0 +1,392 @@
...
+	if err != nil {
+		t.Fatal(err)
+	}
+	// strip the intermediate signers which are included
+	servingCertConfig.Certs = servingCertConfig.Certs[:1]
+	servingPEMCertByte, servingPEMKeyBytes, err := servingCertConfig.GetPEMBytes()
+	if err != nil {
+		t.Fatal(err)
+	}
+	t.Log(string(servingPEMCertByte))
```
Comment (slaskawi) [inline pkg/crypto/cert_accept_reject_test.go:72]: We should probably remove this or print other certs out as well. I'm fine with either way.

Comment (s-urbaniak) [inline pkg/crypto/cert_accept_reject_test.go:72]: leftover?

**Review by slaskawi — APPROVED** [review]
/lgtm

**Pull request merged**